source/includes/debt-settlement.md.erb

```diff
@@ -588,11 +588,12 @@ Content-Type: application/json
 ### Description
 Adds repayments to investments from a specified flexible payments wallet.
 
- - On receiving the API call, the system will reserve money in the wallet and start the distribution process provided the "sourceWalletId" has enough funds.
+ - On receiving the API call, the system will reserve money in the wallet and start the distribution process provided the `sourceWalletId` has enough funds.
  - There is no need to manually move money from a wallet to the receiving account.
- - The system will publish a <a href="/#webhooks-investment_repayment_instruction_received">INVESTMENT_REPAYMENT_INSTRUCTION_RECEIVED</a> webhook once the repayment batch has been accepted.
- - The system will then publish "n" webhooks of type <a href="/#webhooks-investment_repayment_clearing">INVESTMENT_REPAYMENT_CLEARING</a>, where "n" is the size of the investorRepayments array, as each repayment in the batch is cleared.
- - Finally the system will publish a <a href="/#webhooks-investment_repayment_batch_complete">INVESTMENT_REPAYMENT_BATCH_COMPLETE</a> when the batch in considered complete. This is a batch process and therefore might be sent later than the last repayment to clear.
+ - The system will publish an <a href="/#webhooks-investment_repayment_instruction_received">INVESTMENT_REPAYMENT_INSTRUCTION_RECEIVED</a> webhook once the repayment batch has been accepted.
+ - The system will then publish "n" webhooks of type <a href="/#webhooks-investment_repayment_clearing">INVESTMENT_REPAYMENT_CLEARING</a>, where "n" is the size of the `investorRepayments` array, as each repayment in the batch is cleared.
+ - Finally the system will publish an <a href="/#webhooks-investment_repayment_batch_complete">INVESTMENT_REPAYMENT_BATCH_COMPLETE</a> webhook when the batch in considered complete. This is a batch process and therefore might be sent later than the last repayment to clear.
+ - These webhooks should provide coverage of all stages of the batch clearing. When using <a href="/#webhooks-investment_repayment_clearing">INVESTMENT_REPAYMENT_CLEARING</a>, it might be relevant to ignore all `CAPITAL_REPAYMENT` and `INTEREST_REPAYMENT` <a href="/#webhooks-investor_funds_received">INVESTOR_FUNDS_RECEIVED</a> webhooks.
```

Comment:
I'd be 100% clear here:

all <a href="/#webhooks-investor_funds_received">INVESTOR_FUNDS_RECEIVED</a> where the `sourceType` is `CAPITAL_REPAYMENT` or `INTEREST_REPAYMENT`